### PR TITLE
handle thread interrupt

### PR DESCRIPTION
**Pull request by service-michaelnielson-taulia**
See: https://github.com/Graylog2/log4j2-gelf/issues/8

I don't think master is up to date, this needs to be merged right?

https://github.com/taulia/log4j2-gelf/pull/1/files

Are there other versions out there with fixes beyond: 1.1.0.taulia

Reviewer/Merger: @livingtocode-taulia

### CODE REVIEW

File: src/main/java/org/graylog2/log4j2/GelfAppender.java [hunk 1]
```diff
@@ -135,7 +135,17 @@ public void append(LogEvent event) {
         }
 
         try {
-            client.send(builder.build());
+          client.send(builder.build());
+        } catch (InterruptedException e){
+          try {
+            boolean sentMessage = client.trySend(builder.build());
+            if (!sentMessage) {
+              throw new AppenderLoggingException("interrupted while attempting to write a log message", e);
```
Comment (livingtocode-taulia) [inline src/main/java/org/graylog2/log4j2/GelfAppender.java:143]: Is this recoverable? What happens in this situation?

Comment (service-michaelnielson-taulia) [inline src/main/java/org/graylog2/log4j2/GelfAppender.java:143]: The log message goes to stdout, previously this was handled by line 150

**Comment by livingtocode-taulia** [comment]
I merged my previous PR and will make a new build that'll be published as `1.1.0taulia2`

**Comment by service-michaelnielson-taulia** [comment]
Thanks!

**Pull request merged**